docs/structured-streaming-kafka-integration.md

```diff
@@ -119,6 +119,122 @@ ds3.selectExpr("CAST(key AS STRING)", "CAST(value AS STRING)")
 </div>
 </div>
 
+### Creating a Kafka Source Batch
+
```

Comment:
It's better to add a sentence here before the codes, such as `If you have a use case that is better suited to batch processing, you can create an Dataset/DataFrame for a defined range of offsets.`